### DISCUSSION TITLE
How do I make a PayShap payment via API

### DISCUSSION THREAD

**Discussion by gontadu** [Investec-Developer-Community/.github#26] [Q&A] [answered]
Looked through the API docs and I can't see how this is done because the faster payment option is just a boolean. Unless it has a default somewhere e.g. below R 500 is always PayShap?

**Comment by programmable-banking-community** [comment, marked as answer]
Hi @gontadu 👋 The guys are still testing the Investec PayShap API integration. As soon as it's in production we'll let you know. Currently, only Beneficiary payments are available via the API (https://developer.investec.com/za/api-products/documentation/SA_PB_Account_Information.)

Thanks for adding your vote to prioritising a PayShap API connection on our roadmap 🧑‍💻.

**Reply by gontadu** [reply]
Do we have a ballpark ETA?

**Reply by programmable-banking-community** [reply]
It depends on any findings from the internal testing first, after which we'll have a better date. The aim is to have it out in the first part of 2025, so Q1/Q2 2025 (tbc).
